Documentation/rkt-vs-other-projects.md

```diff
@@ -51,6 +52,14 @@ As rkt does not have a centralized daemon it can also be easily integrated with
 [runc-github]: https://github.com/opencontainers/runc
 [oci-spec-github]: https://github.com/opencontainers/specs
 
+## rkt vs containerd
+
+[containerd][containerd-github] is a deamon to control runC. It has a command-line tool called `ctr` which is used to interact with the containerd daemon. This makes the containerd process model similar to that of the Docker process model, illustrated above.
```

Comment:
daemon
one sentence per line please, in keeping with the rest of the doc

containerd-github doesn't point anywhere